src/server/sockets/setup-connection.js

```diff
@@ -30,4 +40,8 @@ const setupConnection = io => socket => {
     .do(bid => io.emit('new_bid_placed', bid))
 }
 
+function emitWatchCount(io, watchCount) {
```

Comment:
You could use `io.sockets.sockets` to get the number of connections like so:
```javascript
const getNumberOfSockets = io => Object.keys(io.sockets.sockets).length
```
then your `emitWatchCount()` signature becomes simpler:
```javascript
function emitWatchCount(io) {                                                                                                          |                                                                                                                                        
  io.emit('watch_count', getNumberOfSockets(io) - 1)                                                                                   |                                                                                                                                        
} 
```